app.js

```diff
@@ -12,10 +16,25 @@ const notFoundMiddleware = require('./middleware/not-found');
 const errorHandlerMiddleware = require('./middleware/error-handler');
 const authenticationMiddleware = require('./middleware/authentication');
 
+app.set('trust proxy', 1);
+app.use(
+  rateLimiter({
+    windowMs: 15 * 60 * 1000,
+    max: 100,
+  })
+);
+app.use(express.json());
+app.use(helmet());
+app.use(cors());
+app.use(xss());
+
 app.use(express.static('./public'));
```

Comment:
Line 31 might need to change to this to get heroku to play nice

app.use(express.static(__dirname + './public'));

Reply:
@loitijoseph forget about my previous comment. 
As we saw, it deployed nicely with the current format and the new one is breaking on my computer.
But as for lines 35 - 37

app.get('/', (req, res) => {
  res.send('Dear Diary API');
});

make sure you comment that out or delete it.

Reply:
Ok. Got it